configs/benchmark/templates/wrk2.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: wrk2-prometheus
+  namespace: {{.Release.Namespace}}
+  labels:
+    jobgroup: wrk2-prometheus
+spec:
+  template:
+    metadata:
+      name: wrk2-prometheus
+      labels:
+        jobgroup: wrk2-prometheus
+        app: wrk2-prometheus
+        custom-affinity: load-generator-node
+    spec:
+      restartPolicy: Never
+      tolerations:
+      - effect: NoSchedule
+        key: load-generator-node
+        operator: Exists
+      volumes:
+      - name: tempfs
+        emptyDir:
+          medium: Memory
+      shareProcessNamespace: true
+      containers:
+      - name: wrk2-prometheus
+        image: quay.io/kinvolk/wrk2-prometheus
+        imagePullPolicy: Always
+        volumeMounts:
+        - name: tempfs
+          mountPath: "/tmpfs"
+        args:
+        - -p
+        {{- if empty .Values.wrk2.serviceMesh }}
+        - "http://pushgateway.{{.Release.Namespace}}:9091/metrics/job/bare-metal/instance/{{.Values.wrk2.app.name}}/run/{{ now | date "20060102150405" }}"
+        {{- else if eq .Values.wrk2.serviceMesh "linkerd" }}
+        - "http://pushgateway.{{.Release.Namespace}}:9091/metrics/job/svmesh-linkerd/instance/{{.Values.wrk2.app.name}}/run/{{ now | date "20060102150405" }}"
+        {{- else if eq .Values.wrk2.serviceMesh "istio" }}
+        - "http://pushgateway.{{.Release.Namespace}}:9091/metrics/job/svmesh-istio/instance/{{.Values.wrk2.app.name}}/run/{{ now | date "20060102150405" }}"
+        {{- else }}
+        {{ fail "Please provide a valid 'wrk2.serviceMesh' name, supported names are: 'linkerd', 'istio'. For no service mesh skip the variable." }}
+        {{- end }}
+        - -c
+        - "{{.Values.wrk2.connections}}"
+        - -d
+        - "{{.Values.wrk2.duration}}"
+        - -r
+        {{- if lt (.Values.wrk2.RPS | int) 400 }}
```

Comment:
Why not? We should be able to set arbitrary values.

Reply:
What do you mean? This is just a check to see if the value is less than 400 if yes then it fails, similar to the way you have it in your script.

Reply:
The script implements the following:
- calculate number of connections/threads by dividing the RPS by 400 (if connections/threads are not specified on the command line)
- if the result of the calculation is 0 then set connections / threads to 1 (because the benchmark will not run with 0 connections / threads)

Reply:
So should I remove this check or keep it? I am confused!

Reply:
Please pardon, I did not mean to confuse. The check is unnecessary because the helm chart - contrary to the script - does not calculate `Values.wrk2.connections`; so `Values.wrk2.connections` can never be accidentally calculated to be 0. The check can be removed.

Reply:
No worries, I was more of confused from my lack of understanding. I think I understand it now. Thanks for clarifying the confusiton Thilo. :-)